grc/azure_software_radio_rest_api.block.yml

```diff
@@ -25,26 +27,26 @@ templates:
 
 documentation: |
 
-    Creates REST endpoints to get flowgrpah status, get/set variables, and trigger callbacks.
+    Creates REST endpoints to get flowgraph status, get/set variables (by triggering their callbacks).
 
         The following routes are supported (no authentication is used):
             - GET http://hostname:<port>/status
-            - GET http://hostname:<port>/config
             - PUT http://hostname:<port>/config
```

Comment:
we can add the POST config route to the docs even if it does the same as PUT at the moment

Reply:
I was worried about causing confusion but I guess we can just say it is functionally equivalent and leave it at that